### PR TITLE
fix config index

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

**Comment by nslater** [comment]
basing this on `nomi/reshuffle-sql-ref` (for the `core` reference) but will re-base to `master` once the `nomi/reshuffle-sql-ref` branch is merged, and then merge this pr

**Comment by nslater** [comment]
see https://github.com/crate/crate/pull/6241

File: blackbox/docs/configuration/index.txt [hunk 1]
```diff
@@ -46,8 +43,12 @@ Settings are applied in the following order:
 Each setting value overwrites any previous value. So, for example, command line
 settings will override configuration file settings.
 
-Topics
-======
+.. NOTE::
+
+   If you're just getting started with a particular part of CrateDB, we
+   recommend you consult the appropriate top-level section of the
```
Comment (christianbader) [inline blackbox/docs/configuration/index.txt]: recommend you to consult ...

Comment (nslater) [reply blackbox/docs/configuration/index.txt]: hmm. I don't think so. sounds wrong to my ear! thx though!

Comment (christianbader) [reply blackbox/docs/configuration/index.txt]: @nslater i guess you are right ;)

**Pull request merged**